client/src/includes/contentMetrics.test.ts

```diff
@@ -0,0 +1,41 @@
+import { getContentMetrics } from './contentMetrics';
+
+describe('getContentMetrics', () => {
+  it('should return correct wordCount and readingTime using Intl.Segmenter', () => {
+    const result = getContentMetrics('en-US', 'This is a test sentence.');
+    expect(result.wordCount).toBe(5);
+    expect(result.readingTime).toBe(0);
+  });
+
+  it('should handle empty text', () => {
+    const result = getContentMetrics('en-US', '');
+    expect(result.wordCount).toBe(0);
+    expect(result.readingTime).toBe(0);
+  });
+
+  it('should handle text with punctuation correctly', () => {
```

Comment:
I think we should aim for more granular test cases that are easier to assess. As-is it’s pretty much impossible to read the contents of this test case and get a sense that the word count is correct.

I’d recommend:

- 5-10 test cases at most.
- At most 1 line of test data per case.
- Below-10 word count per test case

Something like:

```js
describe.each`
  text                                               | count
  ${"¿Donde esta la biblioteca?"}                    | ${9}
  ${"It’s lots. Of; Punctuation"}                    | ${8}
  ${"האהבה היא אוקיינוס שאין לו התחלה ואין לו סוף."} | ${7}
  ${"元気です、ありがとう。あなたは？"}              | ${5}
`("getWordCount", ({ text, result }) => {
```